gobblin-runtime/src/main/java/gobblin/runtime/AbstractJobLauncher.java

```diff
@@ -331,6 +321,7 @@ protected void postProcessTaskStates(List<TaskState> taskStates) {
    * Subclasses can override this method to do whatever processing on the {@link JobState} and its
    * associated {@link TaskState}s, e.g., aggregate task-level metrics into job-level metrics.
    */
+  @SuppressWarnings("deprecation")
```

Comment:
Can you explain in Javadoc why this is deprecated and what should the caller use instead.